.gitignore

```diff
@@ -8,6 +8,7 @@ lib
 distribute.egg-info
 setuptools.egg-info
 .coverage
+test_*
```

Comment:
This change doesn't seem right. I expect `test_*` to be tracked by git. Why the change?

Reply:
I don't remember, must be a leftover from something.

Reply:
```suggestion
```